dashboard/yarn.lock

```diff
@@ -382,6 +382,14 @@ accepts@~1.3.4:
     mime-types "~2.1.16"
```

Comment:
why did you need to update the dependencies? (I am asking because the package.json remains the same)

Reply:
Good catch, this was because in previous versions of the patch I upgraded react-scripts-ts and for some reason yarn.lock did not downgrade. I'll put it back.

Thank you!